src/test/java/org/apache/maven/plugins/assembly/utils/FilterUtilsTest.java

```diff
@@ -183,21 +184,18 @@ public void testFilterProjects_ShouldNotRemoveProjectNotIncludedAndNotExcluded()
     @Test
     public void testTransitiveScopes()
     {
-        Assert.assertThat( FilterUtils.newScopeFilter( "compile" ).getIncluded(),
-                           Matchers.containsInAnyOrder( "compile", "provided", "system" ) );
+        assertThat( FilterUtils.newScopeFilter( "compile" ).getIncluded(),
```

Comment:
Perhaps unused `import org.junit.Assert;` at line 42/43 could be removed then?

Reply:
done